components/sensor/scd30.rst

```diff
@@ -0,0 +1,73 @@
+SCD30 CO<sub>2</sub>, Temperature and Relative Humidty Sensor
+=============================================================
+
+.. seo::
+    :description: Instructions for setting up SCD30 CO<sub>2</sub> Temperature and Relative Humidty Sensor
```

Comment:
```suggestion
    :description: Instructions for setting up SCD30 CO₂ Temperature and Relative Humidty Sensor
```

sphinx will probably scape the <sub> here, use unicode instead to avoid the issue.

Reply:
I've replaced the tagged subs with the unicode symbol.